assets/theme-css/sphinx-design/_dropdown.scss

```diff
@@ -79,6 +103,61 @@ details.sd-dropdown {
     visibility: hidden;
   }
 
+  // Chevron transitions
+  .sd-summary-chevron-right i,
+  .sd-summary-chevron-down i {
+    display: inline-block;
+    transform-origin: center;
+    opacity: 0.6;
+  }
+
+  // The chevron rotation animations are applied to
+  // the icon inside the dropdown title div
+  &[open] > .sd-summary-title .sd-summary-chevron-right i {
+    transform: rotate(90deg);
+    animation: rotate-to-90 0.25s ease-in-out;
+  }
+
+  &[open] > .sd-summary-title .sd-summary-chevron-down i {
+    transform: rotate(-180deg);
+    animation: rotate-to-negative-180 0.25s ease-in-out;
+  }
+
+  &:not([open]) > .sd-summary-title .sd-summary-chevron-right i {
+    transform: rotate(0deg);
+    animation: rotate-to-0-from-90 0.25s ease-in-out;
+  }
+
+  &:not([open]) > .sd-summary-title .sd-summary-chevron-down i {
+    transform: rotate(0deg);
+    animation: rotate-to-0-from-negative-180 0.25s ease-in-out;
+  }
+
+  > .sd-summary-title:hover .sd-summary-chevron-right i,
+  > .sd-summary-title:hover .sd-summary-chevron-down i {
+    opacity: 1;
+  }
+
+  // Combined transforms for each state with hover. These cover
+  // the cases where the chevron is rotated, say, when the dropdown
+  // is open or if the chevron starts rotated (e.g. in the down-up
+  // state).
+  &:not([open]) > .sd-summary-title:hover .sd-summary-chevron-right i {
+    transform: scale(1.1);
+  }
+
+  &:not([open]) > .sd-summary-title:hover .sd-summary-chevron-down i {
+    transform: scale(1.1);
+  }
+
+  &[open] > .sd-summary-title:hover .sd-summary-chevron-right i {
+    transform: rotate(90deg) scale(1.1);
+  }
+
+  &[open] > .sd-summary-title:hover .sd-summary-chevron-down i {
+    transform: rotate(-180deg) scale(1.1);
+  }
```

Comment:
In the earlier commit I pushed today, I changed this to `.sd-summary-title:hover` as we want to increase the chevron sizes only when hovering over the dropdown bars and not when hovering over the content inside them (this is the same as PST).

In particular, these lines address the cases where you hover over the chevrons and click on them – we don't want the chevrons to transition from a 1.1x scale to a 1.0x scale while rotating and then back to a 1.1x scale, but rather stay at a 1.1x scale throughout the transition.